assignments/arrays.js

```diff
@@ -63,32 +63,65 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let forgottenCar = inventory[32];
+console.log("Car 33 is a " + forgottenCar.car_year + " " + forgottenCar.car_make + " " + forgottenCar.car_model + ".");
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[49];
+console.log(lastCar.car_make + " " + lastCar.car_model);
+
+//{"id":50,"car_make":"Lincoln","car_model":"Town Car","car_year":1999}];
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (let i = 0; i < inventory.length; i++) {
+    let carID = inventory[i]; 
+    carModels.unshift(carID.car_model);
+    carModels.sort();
+}
+
+console.log(carModels);
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for (let i = 0; i < inventory.length; i++) {
+    let carAnnum = inventory[i]; 
+    carYears.unshift(carAnnum.car_year);
+}
+console.log(carYears);
```

Comment:
Awesome job using the unshift method here. (something I FORGOT and had to look up) Thanks for the refresher on that. :)  I love that your trying different things!